docs/source/Setup/Installation.md

```diff
@@ -13,116 +13,114 @@ You can also  [clone Evennia from github](./Installation-Git.md)  or use [docker
+The resultant folder contains all the empty templates and default settings needed to start the Evennia server.
 
-## Start the new game
+## Start the New Game
 
-    cd mygame 
-    evennia migrate
+First, create the default database (Sqlite3):
 
-This will create the default database (Sqlite3). The database file ends up as `mygame/server/evennia.db3`. If you ever want to start from a fresh database, just delete this file and re-run `evennia migrate` again.
+	cd mygame
+	evennia migrate
 
-    evennia start
+The resulting database file is created in `mygame/server/evennia.db3`. If you ever want to start from a fresh database, just delete this file and re-run the `evennia migrate` command.
 
-Set your user-name and password when prompted. This will be the "god user" or "super user" in-game.  The email is optional.
+Next, start the Evennia server with:
 
-> You can also [automate](./Installation-Non-Interactive.md) the creation of the super user.
+	evennia start
 
-If all went well, the server is now up and running. Point a legacy MUD/telnet client to `localhost:4000` or a web browser at [http://localhost:4001](http://localhost:4001) to play your new (if empty) game! 
+When prompted, enter a username and password for the in-game "god" or "superuser." Providing an email address is optional.
 
-Log in as a new account or use the superuser you just created.
+> You can also [automate](./Installation-Non-Interactive.md) creation of the superuser.
 
+If all went well, your new Evennia server is now up and running! To play your new -- albeit empty -- game, point a legacy MUD/telnet client to `localhost:4000` or a web browser to [http://localhost:4001](http://localhost:4001). You may log in as a new account or use the superuser account you created above.
 
-## Restarting and stopping
+## Restarting and Stopping
 
 
-You can restart the server without disconnecting players:
+You can restart the server (without disconnecting players) by issuing:
 
-    evennia restart
+	evennia restart
 
-To do a full stop and restart (will disconnect players):
+And, to do a full stop and restart (with disconnecting players) use:
 
-    evennia reboot
+	evennia reboot
 
-Full stop of the server (use `evennia start` to restart):
+A full stop of the server (use `evennia start` to restart) is achieved with:
 
-    evennia stop
+	evennia stop
 
-See [Server start-stop-reload](./Running-Evennia.md) page for more details.
+See the [Server start-stop-reload](./Running-Evennia.md) documentation page for details.
 
-## See server logs
+## View Server Logs
 
-Log files are in `mygame/server/logs`. You can tail them live with
+Log files are located in `mygame/server/logs`. You can tail the logging in real-time with:
 
-    evennia --log
+	evennia --log
 
-or
+or just:
 
-    evennia -l
+	evennia -l
 
-Stop viewing the log by pressing `Ctrl-C` (`Cmd-C` for Mac). 
+Pressing `Ctrl-C` (`Cmd-C` for Mac) to stop viewing the live log. 
```

Comment:
Suggesting `Press` rather than `Pressing` here, to be grammatically correct.

Reply:
Yep. Typo'd it.